src/ducks/reimbursements/ReimbursementsPage.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { TransactionsPage } from 'ducks/transactions'
+
+class ReimbursementsPage extends React.PureComponent {
```

Comment:
J'ai l'impression que tu peux supprimer ce fichier et faire comme [ici](https://github.com/cozy/cozy-banks/pull/931/files#diff-74afc5ac3c5c28eb8d1c3d5227c062c9R42).

Reply:
I refactored the code to export a `TransactionsPageWithBackButton` that I use in both placed :+1: